internal/rsync/rsync.go

```diff
@@ -126,6 +126,11 @@ func rsyncArguments(ctx context.Context, cfg conf.Config, args args.Config) []st
 	if args.Filter != "" {
 		argv = append(argv, "--filter", fmt.Sprint(args.Filter))
 	}
+	if len(args.Exclude) > 0 {
```

Comment:
This looks like another case where the "if" might be redundant, would it work fine just with the "for ... range" loop?

Reply:
Ah, yes.